packages/payment-plugin/e2e/mollie-payment.e2e-spec.ts

```diff
@@ -0,0 +1,196 @@
+import { PaymentStatus } from '@mollie/api-client';
+import { DefaultLogger, LogLevel, mergeConfig } from '@vendure/core';
+import {
+    createTestEnvironment,
+    E2E_DEFAULT_CHANNEL_TOKEN,
+    registerInitializer,
+    SimpleGraphQLClient,
+    SqljsInitializer,
+    testConfig,
+    TestServer,
+} from '@vendure/testing';
+import nock from 'nock';
+import fetch from 'node-fetch';
+import path from 'path';
+
+import { initialData } from '../../../e2e-common/e2e-initial-data';
+import { MolliePlugin } from '../src/mollie';
+import { molliePaymentHandler } from '../src/mollie/mollie.handler';
+import { CREATE_PAYMENT_METHOD, GET_CUSTOMER_LIST } from './graphql/admin-queries';
+import { CreatePaymentMethod, GetCustomerList, GetCustomerListQuery } from './graphql/generated-admin-types';
+import { ADD_ITEM_TO_ORDER, ADD_PAYMENT, GET_ORDER_BY_CODE } from './graphql/shop-queries';
+import {
+    AddItemToOrder,
+    AddPaymentToOrder,
+    GetOrderByCode,
+    TestOrderFragmentFragment,
+} from './graphql/generated-shop-types';
+import { proceedToArrangingPayment, refundOne } from './payment-helpers';
+import { TEST_SETUP_TIMEOUT_MS } from '../../../e2e-common/test-config';
+
+describe('Mollie payments', () => {
+    const mockData = {
+        host: 'https://my-vendure.io',
+        redirectUrl: 'https://my-storefront/order',
+        apiKey: 'myApiKey',
+        methodCode: `mollie-payment-${E2E_DEFAULT_CHANNEL_TOKEN}`,
+        mollieResponse: {
+            id: 'tr_mockId',
+            _links: {
+                checkout: {
+                    href: 'https://www.mollie.com/payscreen/select-method/mock-payment',
+                },
+            },
+        },
+    };
+    let shopClient: SimpleGraphQLClient;
+    let adminClient: SimpleGraphQLClient;
+    let server: TestServer;
+    let started = false;
+    let customers: GetCustomerListQuery['customers']['items'];
+    let order: TestOrderFragmentFragment;
+    beforeAll(async () => {
+        registerInitializer('sqljs', new SqljsInitializer('__data__'));
```

Comment:
Import the `testConfig` constant not from `@vendure/testing`, but from `'../../../e2e-common/test-config';`. Then this call to `registerInitializer` should not be needed, because all initializers for the supported DBs will be implicitly registered. Actually this _should_ already be the case since you are already importing `TEST_SETUP_TIMEOUT_MS` from that file.

Reply:
Also fixed :ok_hand: